docs/src/commands/build.md

```diff
@@ -99,7 +99,7 @@ wasm-pack build --target nodejs
 
 ## Scope
 
-The init command also accepts an optional `--scope` argument. This will scope
+The `build` command also accepts an optional `--scope` argument. This will scope
```

Comment:
I didn't test that you can actually pass `--scope` to `wasm-pack build` but this would seem to be the correct change here (to be symmetrical with other stuff on this page and to match the example command below)